src/components/Comments/WriteComment.js

```diff
@@ -259,5 +289,9 @@ export const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: 12,
     color: COLORS.blackgrey
+  },
+  isViewOnlyIcon: {
+    width: normalize(32),
+    height: normalize(32)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new style `isViewOnlyIcon` has been added. This seems to be used for the send comment icon when the component is in view-only mode.


<!-- This is an auto-generated comment by OSS CodeRabbit -->